src/lib/database/schemas/Version1Schema.ts

```diff
@@ -93,6 +95,6 @@ export async function update(_tx: Transaction) {
 
 export const schema = {
   tricks: "[id+trickStatus], technicalName, alias, establishedBy, yearEstablished, startPosition, endPosition, difficultyLevel, description, *recommendedPrerequisites, tips, *variationOf, showInSearchQueries, dateAddedEpoch, videos",
-  combos: "[id+comboStatus], alias, establishedBy, yearEstablished, *tricks, description, tips, dateAddedEpoch, videos",
+  combos: "[id+comboStatus], name, establishedBy, yearEstablished, *tricks, description, tips, dateAddedEpoch, videos",
```

Comment:
technically, this would warrant a DB Version Bump, but given that there is no release version yet, we can ignore that :D